docs-beta/content/quickstart/_index.md

```diff
@@ -0,0 +1,466 @@
+  "namespaceName": "default",
+  "retentionTime": "12h"
+}'
+```
+
+{{% /tab %}}
+{{% tab name="Output" %}}
+
+```json
+{
+  "namespace": {
+    "registry": {
+      "namespaces": {
+        "default": {
+          "bootstrapEnabled": true,
+          "flushEnabled": true,
+          "writesToCommitLog": true,
+          "cleanupEnabled": true,
+          "repairEnabled": false,
+          "retentionOptions": {
+            "retentionPeriodNanos": "43200000000000",
+            "blockSizeNanos": "1800000000000",
+            "bufferFutureNanos": "120000000000",
+            "bufferPastNanos": "600000000000",
+            "blockDataExpiry": true,
+            "blockDataExpiryAfterNotAccessPeriodNanos": "300000000000",
+            "futureRetentionPeriodNanos": "0"
+          },
+          "snapshotEnabled": true,
+          "indexOptions": {
+            "enabled": true,
+            "blockSizeNanos": "1800000000000"
+          },
+          "schemaOptions": null,
+          "coldWritesEnabled": false,
+          "runtimeOptions": null
+        }
+      }
+    }
+  },
+  "placement": {
+    "placement": {
+      "instances": {
+        "m3db_local": {
+          "id": "m3db_local",
+          "isolationGroup": "local",
+          "zone": "embedded",
+          "weight": 1,
+          "endpoint": "127.0.0.1:9000",
+          "shards": [
+            {
+              "id": 0,
+              "state": "INITIALIZING",
+              "sourceId": "",
+              "cutoverNanos": "0",
+              "cutoffNanos": "0"
+            },
+            …
+            {
+              "id": 63,
+              "state": "INITIALIZING",
+              "sourceId": "",
+              "cutoverNanos": "0",
+              "cutoffNanos": "0"
+            }
+          ],
+          "shardSetId": 0,
+          "hostname": "localhost",
+          "port": 9000,
+          "metadata": {
+            "debugPort": 0
+          }
+        }
+      },
+      "replicaFactor": 1,
+      "numShards": 64,
+      "isSharded": true,
+      "cutoverTime": "0",
+      "isMirrored": false,
+      "maxShardSetId": 0
+    },
+    "version": 0
+  }
+}
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+Placement initialization can take a minute or two. Once all the shards have the `AVAILABLE` state, the node has finished bootstrapping, and you should see the following messages in the node console output.
+
+<!-- TODO: Fix these timestamps -->
+
+```shell
+{"level":"info","ts":1598367624.0117292,"msg":"bootstrap marking all shards as bootstrapped","namespace":"default","namespace":"default","numShards":64}
+{"level":"info","ts":1598367624.0301404,"msg":"bootstrap index with bootstrapped index segments","namespace":"default","numIndexBlocks":0}
+{"level":"info","ts":1598367624.0301914,"msg":"bootstrap success","numShards":64,"bootstrapDuration":0.049208827}
+{"level":"info","ts":1598367624.03023,"msg":"bootstrapped"}
+```
+
+You can check on the status by calling the _placement_ endpoint:
```

Comment:
Should we also put that you can hit the /namespace endpoint to see all configured namespaces? also, btw if you include `?debug=1` in the /namespace request, the numbers you get back for retention period, block size, etc. are in standard units and not just nanoseconds, which makes it a lot easier to grok.

Reply:
Yes, it will return all namespaces. There's no way (currently) to return just one namespace.